vhdbuilder/packer/install-dependencies.sh

```diff
@@ -194,28 +194,49 @@ if [[ $OS == $UBUNTU_OS_NAME ]]; then
   done
 fi
 
+if [[ $OS == $UBUNTU_OS_NAME && $(isARM64) != 1 ]]; then  # no ARM64 SKU with GPU now
+  gpu_action="copy"
+  if grep -q "fullgpu" <<< "$FEATURE_FLAGS"; then
+    # force older driver on dedicated vhd to avoid build failures.
+    export NVIDIA_DRIVER_IMAGE_TAG="470.57.02-v0.0.2-rev17-sha-ff708e"
```

Comment:
gross but effective to keep gpu vhd pinned to old version

Reply:
I guess with this change we might (?) be able to bump the gpu vhd anyway, since it will be able to egress to MCR? not sure what RP e2e will do...